### PR TITLE
Bump zwave-js and zwave-js-server

### PR DESCRIPTION
**Pull request by raman325**
zwave-js: 8.0.3
https://github.com/zwave-js/node-zwave-js/releases/tag/v8.0.1
https://github.com/zwave-js/node-zwave-js/releases/tag/v8.0.2
https://github.com/zwave-js/node-zwave-js/releases/tag/v8.0.3
zwave-js-server: 1.9.0
https://github.com/zwave-js/zwave-js-server/releases/tag/1.9.0

### CODE REVIEW

**Review by frenck — APPROVED** [review]
LGTM ✅

**Review by frenck — CHANGES_REQUESTED** [review]
Oh, add-on config version bump is missing!

**Comment by raman325** [comment]
doh thanks!

**Review by frenck — APPROVED** [review]

File: zwave_js/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.1.30
+
+- Bump Z-Wave JS to 8.0.2
```
Comment (MartinHjelmare) [inline zwave_js/CHANGELOG.md:5]:
```suggestion
- Bump Z-Wave JS to 8.0.3

```

**Pull request merged**